src/Service/ResumeService.php

```diff
@@ -91,8 +91,6 @@ public function __construct(
      *
      * @param Auth $auth An authentication tracker.
      *
-     * @return bool Whether or not a session still exists.
```

Comment:
I think we should use `@return null` . See https://github.com/auraphp/Aura.Auth/blob/f83060d3004af3777f0fc2b6c6672174e6133f7d/src/Service/LogoutService.php#L68